src/main/antlr4/Manifold.g4

```diff
@@ -89,14 +92,23 @@ lvalue:
   ;
 
 // TODO: declarations as expressions
-expression: /* declaration | */ rvalue;
+expression:
+  rvalue
+  // | declaration
+  ;
+
+statement:
+    expression #ExpressionStatment
+    | 'import' STRING_VALUE #ImportStatement
+    ;
+
+STATEMENT_TERMINATOR: ';';
```

Comment:
> Statements are used because in the imperative paradigm, there are many common operations which don't have a useful result value, and because the notion of sequential instructions (rather than calculations) fits that paradigm quite well. If a large percentage of your program is mutating state, rather than calculating new values, it doesn't make sense to require producing a value (what's the result of a for loop?). In contrast, when your entire paradigm (FP) is built around the calculation of values, outliers that don't have a result value don't warrant an exception: Instead you give them a sentinel result that doesn't mean a thing.

http://programmers.stackexchange.com/questions/179624/could-a-programming-language-work-as-well-without-statements